.github/workflows/pr-build.yml

```diff
@@ -1,18 +1,21 @@
 # This workflow will do a clean install of node dependencies, build the source code, and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Node.js CI
+name: Build PR
+
+# If adding a step that requires SECRETS or WRITE ACCESS be sure to have dependabot skip it.
+# > if: ${{ github.actor != 'dependabot[bot]' }}
 
 on:
   pull_request:
     branches: [main]
 
 jobs:
   # This job will:
-  #   * deploy a draft every time there is a pull request created or synchronized that is not on main branch
-  #   * comment on that pull request with the deploy URL
+  #   * build each package
+  #   * run unit tests
+  #   * run the linter
   build:
-    if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
have to take out the token for the checkout code step for dependabot to run.

Reply:
I removed it. Thanks for the heads up